automation/pr-to-meta-resin.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# PR a supervisor release to meta-resin
+#
+# It clones meta-resin and opens a PR changing the supervisor version to $TAG.
+# If a meta-resin folder exists, it assumes it has a meta-resin git repo.
+#
+# If $PR_1X is "true", an additional PR for 1.X will be created.
+#
+# Requires $GITHUB_USERNAME and $GITHUB_PASSWORD to push and create the pull-request.
+# Requires $TAG to be set to the supervisor version to use.
+#
+
+set -e
+
+THIS_FILE=$0
+if [ -z "$TAG" ]; then
+	cat $THIS_FILE | awk '{if(/^#/)print;else exit}' | tail -n +2 | sed 's/\#//'
+	exit 1
+fi
+
+if [[ -z "$GITHUB_PASSWORD" || -z "$GITHUB_USERNAME" ]]; then
+	echo "GITHUB_USERNAME and GITHUB_PASSWORD are required"
+	exit 1
+fi
+
+USER=${USER:-`whoami`}
```

Comment:
```bash
USER=${USER:-$(whoami)}
```
is preferable to `` ` `` because it allows for nesting and handles a bunch of other situations without surprises (it's not necessary in this case, but it's still a good habit to get into)